docs/core/linux-prerequisites.md

```diff
@@ -581,12 +587,13 @@ To install .NET Core 2.x for supported SUSE Linux Enterprise Server and OpenSUSE
 
 Download and follow the .NET Core 2.1 installation instructions for supported Alpine Linux versions (64 bit) at the following links:
 
-* .NET Core Runtime 2.1.2 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/runtime-2.1.2)
-* .NET Core SDK 2.1.302 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/sdk-2.1.302)
-* .NET Core Runtime 2.1.1 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/runtime-2.1.1)
-* .NET Core SDK 2.1.301 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/sdk-2.1.301)
-* .NET Core Runtime 2.1.0 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/runtime-2.1.0)
-* .NET Core SDK 2.1.300 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/sdk-2.1.300)
+* .NET Core Runtime 2.1.2 [download link](https://www.microsoft.com/net/download/thank-you/dotnet-runtime-2.1.2-linux-x64-alpine-binaries)
```

Comment:
@leecow we don't have a similar page for alpine like the other distributions have?

Reply:
Nope

Reply:
That's correct. At the moment it's only referenced in the 'Other downloads' table at the bottom of https://www.microsoft.com/net/download, under the Linux tab.